webpack.config.js

```diff
@@ -1,17 +1,16 @@
 const path = require('path');
-const glob = require('glob');
 
 const isProduction = process.env.NODE_ENV == "production";
 
 const config = {
-  entry: ["./custom/styles/styles.scss"],
-  output: {
-    path: path.resolve(__dirname, "./public/scripts"),
-    filename: 'index.js'
+  entry: {
+    style: "./custom/styles/style.scss",
+    error: "./custom/styles/errors.scss",
+    main: "./scripts/index.ts"
   },
-  devServer: {
-    open: true,
-    host: "localhost",
+  output: {
+    path: path.resolve(__dirname, "./public/bundle"),
+    filename: '[name]-bundle.js'
```

Comment:
outputting our CSS as a .js file doesn't seem quite right .... can you say more about this?

I'm poking around online and it seems like this is where people typically use plugins like MiniCssExtractPlugin? I haven't dove deep on webpack though

Reply:
I think what happens is webpack bundles everything, JavaScript, CSS, etc into a single JS file.  I modified it to output multiple files here because `errors.scss` has conflicting styles with `style.scss`.  Let me look at MiniCssExtractPlugin.

Reply:
Added `MiniCssExtractPlugin`